pkg/client/clientset_generated/release_1_3/typed/extensions/v1beta1/extensions_client.go

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 package v1beta1
 
 import (
+	fmt "fmt"
```

Comment:
That's my import code-- it does that because it's too dumb to tell if the package name matches the path name. Could be fixed with a small amount of effort.